src/substrate/handler/l1sync.ts

```diff
@@ -88,6 +92,18 @@ async function verify(
        * 3. get hash from db, and check if it is pending in eth
        */
       console.log("Current Verifier Version:" + vid);
+
```

Comment:
Never use the process.exit.
We should use throw error

Reply:
Again the bridge if for the whole bridge contract. 
getEstimatedGasFee is for verify API's gas fee only so need change the name